ts/sdk/Cargo.toml

```diff
@@ -11,6 +11,7 @@ default = []
 
 [dependencies]
 borsh = { workspace = true }
+bs58-fixed-wasm = { workspace = true, features = ["js"]}
```

Comment:
need to set `features = ["js"]` for `tsify-next` dependency as well, check if this affects .wasm size